ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml

```diff
@@ -30,45 +30,6 @@ tests:
       IMAGE_REPO: grc-policy-framework-tests
       REGISTRY_ORG: stolostron
     workflow: ocm-ci-image-mirror
-- as: test-e2e-kind
-  steps:
-    env:
-      AWS_CREDENTIALS_SECRET: ocm-grc-aws-kind
-      AWS_REGION: us-east-1
-    test:
-    - as: kind-test
-      commands: |
-        echo "IMAGE_REF=$IMAGE_REF"
-        ./build/run-test-kind-prow.sh
-      dependencies:
-      - env: IMAGE_REF
-        name: grc-policy-framework-tests
-      from: src
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    workflow: ocm-e2e-kind
-- as: test-e2e-kind-deployonhub
-  steps:
-    env:
-      AWS_CREDENTIALS_SECRET: ocm-grc-aws-kind
-      AWS_REGION: us-east-1
-    test:
-    - as: kind-test
-      commands: |
-        echo "IMAGE_REF=$IMAGE_REF"
-        export deployOnHub="true"
-        ./build/run-test-kind-prow.sh
-      dependencies:
-      - env: IMAGE_REF
-        name: grc-policy-framework-tests
-      from: src
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    workflow: ocm-e2e-kind
 - as: test-e2e-grc-framework
   run_if_changed: ^test/integration/.*
```

Comment:
@dhaiducek is this the bit you meant?

Reply:
Haha--yeah. It's definitely okay the way it is. 😄